src/styles/components/_home-search-box.scss

```diff
@@ -1,18 +1,65 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  display: flex;
+  width: 80%;
 
-  &__button {
-    @include button-darken-effect($picton-blue);
+  background-color: $light;
+  box-shadow: rem(5px 10px 20px) 0px box-shadow-opacity(0.16);
+
+  @include respond-above('md') {
+    width: 90%;
+  }
 
-    text-decoration: none;
+  &__input {
+    width: 100%;
+    box-sizing: border-box;
     padding: rem(5px 10px);
-    border-radius: rem(3px);
-    background-color: $picton-blue;
+
+    color: $dove-gray;
+
+    border: none;
+    font-size: rem(14px);
+    line-height: 1.23;
```

Comment:
Css ordering 🧐
```
    box-sizing: border-box;
    width: 100%;
    padding: rem(5px 10px);
    border: none;

    color: $dove-gray;
    font-size: rem(14px);
    line-height: 1.23;
```

Reply:
So, I think `color` would be separated, right? regarding [CSS ordering rule](https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/).

```
box-sizing: border-box;
    width: 100%;
    padding: rem(5px 10px);
    border: none;

    color: $dove-gray;

    font-size: rem(14px);
    line-height: 1.23;
```